ste/sender-azureFile.go

```diff
@@ -184,6 +177,24 @@ func (u *azureFileSenderBase) Prologue(state common.PrologueState) (destinationM
 		},
 		u.fileOrDirURL,
 		u.jptm.GetForceIfReadOnly())
+
+	if strErr, ok := err.(azfile.StorageError); ok && (strErr.ServiceCode() == azfile.ServiceCodeParentNotFound || strErr.ServiceCode() == azfile.ServiceCodeShareNotFound) {
+		// Create the parent directories of the file. Note share must be existed, as the files are listed from share or directory.
```

Comment:
this is a bit confusing as one of the conditions was "strErr.ServiceCode() == azfile.ServiceCodeShareNotFound"

Reply:
Yeah, you're right. I've removed it.